src/index.html

```diff
@@ -17,16 +17,28 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar top-search-bar"
+      data-qa="big"
+    >
+      <input
+        class="search-bar-input search-bar-input--top"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
```

Comment:
The placeholder text is identical for both input fields. It's better to provide unique placeholder text for each field to guide the user.